gemfiles/activesupport_7_0.gemfile

```diff
@@ -0,0 +1,9 @@
+# This file was generated by Appraisal
```

Comment:
Is there a missing change to `Appraisals` here?

Reply:
Sure is, I didn't actually generate it with Appraisal because it's a pain to make work. But, I should do so.

Reply:
Done